executor/src/witgen/processor.rs

```diff
@@ -61,14 +64,17 @@ impl<'a, 'b, T: FieldElement> Processor<'a, 'b, T> {
                 UnknownStrategy::Zero,
             );
             for identity in self.identities {
+                println!("Identity: {}", identity);
                 self.identity_processor
-                    .process_identity(identity, &row_pair)?;
+                    .process_identity(identity, &row_pair)
+                    .unwrap();
```

Comment:
Why can we do that now?

Reply:
Oops, shouldn't have been committed, also the change above.

I think I'll just undo all changes to this file, as #528 is already queued to be merged.